sql/npc_list.sql

```diff
@@ -2865,175 +2865,164 @@ INSERT INTO `npc_list` VALUES (16924814,'Mystic_Retriever','Mystic Retriever',13
-INSERT INTO `npc_list` VALUES (16929241,'Particle_Gate','',0,-620.000,78.640,-159.840,0,40,40,8,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-INSERT INTO `npc_list` VALUES (16929242,'Particle_Gate','',0,299.949,-161.560,-479.840,0,40,40,8,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-INSERT INTO `npc_list` VALUES (16929243,'Particle_Gate','',0,-20.080,-1.960,-520.000,0,40,40,8,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-INSERT INTO `npc_list` VALUES (16929244,'Particle_Gate','',0,-239.990,-161.190,-500.020,0,40,40,8,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-INSERT INTO `npc_list` VALUES (16929245,'Particle_Gate','',0,-580.000,-1.450,-560.370,0,40,40,8,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-INSERT INTO `npc_list` VALUES (16929246,'Particle_Gate','',0,540.000,-1.390,-520.000,0,40,40,8,0,0,0,3,0x0200000000000000000000000000000000000000,0,'COP',0);
-
+INSERT INTO `npc_list` VALUES (16929188,'Matter_Diffusion_Module','Matter Diffusion Module',0,580.000,-2.375,104.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929189,'Matter_Diffusion_Module','Matter Diffusion Module',0,-620.000,77.500,376.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929190,'Matter_Diffusion_Module','Matter Diffusion Module',0,-620.000,-2.500,96.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929191,'Matter_Diffusion_Module','Matter Diffusion Module',0,-620.000,77.500,-184.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929192,'Matter_Diffusion_Module','Matter Diffusion Module',0,-580.000,-2.500,-584.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929193,'Matter_Diffusion_Module','Matter Diffusion Module',0,-264.000,-162.500,-500.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929194,'Matter_Diffusion_Module','Matter Diffusion Module',0,-20.000,-2.500,-544.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929195,'Matter_Diffusion_Module','Matter Diffusion Module',0,300.000,-162.500,-504.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929196,'Matter_Diffusion_Module','Matter Diffusion Module',0,540.000,-2.500,-544.000,1,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16929197,'_110','',0,340.000,72.000,400.000,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
```

Comment:
could use the empty unused string field to identify it by, if the special name is really required.

Reply:
No need to click or identify these, just using the IDs to set animation for doors.